addons/website_sale/static/src/js/website_sale.js

```diff
@@ -830,6 +840,9 @@ publicWidget.registry.websiteSaleCart = publicWidget.Widget.extend({
 publicWidget.registry.websiteSaleCarouselProduct = publicWidget.Widget.extend({
     selector: '#o-carousel-product',
     disabledInEditableMode: false,
+    events: {
+        'wheel .o_carousel_product_indicators': '_onMouseWheel',
+    },
```

Comment:
I wanted it to be cropped to indicate to the user that there are more images ...
For me it is not strange but I am open to proposals ;)

Reply:
"Strange" is maybe too much, just "not perfect" to me. If it was in modal (or in any box with borders I mean), that would be normal. Anyways, I'll let the majority's opinion decide, it's ok to me, I just feel it could be better, not sure how. Maybe blurring the left/right when an image is cropped? :thinking: